actions/websocket_actions.test.jsx

```diff
@@ -118,6 +119,16 @@ jest.mock('actions/views/rhs', () => ({
     }),
 }));
 
+describe('handleEvent', () => {
+    test('should dispatch channel updated event properly', () => {
+        const msg = {event: SocketEvents.CHANNEL_UPDATED};
+
+        handleEvent(msg);
+
+        expect(store.dispatch).toHaveBeenCalled();
```

Comment:
HMM i wonder if this is testing anything really, all this confirms is something was called, but in reality i could go in and change dispatch to invoke anything else and it would be pass the test, i wonder if it's worth exporting the handler for this? just a thought. Would've been more  valuable if we could assert the function it was invoked, which i think might be harder since it's a thunk action.

Reply:
Yeah, I would've liked to test what it got called with, but I couldn't come up with a way to do it since there's no way to mock `handleChannelUpdatedEvent` here. I mostly cared that `dispatch` gets called in the first place though since that's what was missing previously. We could test that fully if it were broken up into multiple files though 😛

Reply:
heh, true. I mean you could assert that `type: hannelTypes.RECEIVED_CHANNEL,` was dispatched, but honestly though my vote is to remove it since it's not really testing the core functionality, but 0/5 on this.

Reply:
I did try to mock `handleChannelUpdatedEvent` but also couldn't find a way to do it.  I'm good with this as is.